### ISSUE TITLE
How to use with Expo?

### ISSUE THREAD

**Issue by DrZoidberg09** [ajakka/react-native-here-explore#9]
Hi there,

the library looks awesome. I have been waiting for it for so long!

However, I dont get it running with Expo.

Is there any hint to to do it?

Thank you so much!

**Comment by ajakka** [comment]
The library does not officially support expo, but I reckon if you do expo prebuild and follow the instructions in the INSTALLATION.md, it could work.

**Comment by DrZoidberg09** [comment]
No, unfortunately it is not. If you run prebuild it does not find the HereMapsExample, only your own project. And if you run pod install (actually should not be needed), it says that it connot find heremaps.

Just as an idea: Would Expo Modules not an easier way to use the native modules from HERE? Both for pure RN and Expo?

**Comment by ajakka** [comment]
That's unfortunate, I will look into Expo framework and see If I can support it.
also, you are welcome if you got experience with Expo and want to contribute.
